package-lock.json

```diff
@@ -14313,6 +14316,12 @@
         "postcss": "^7.0.14"
       }
     },
+    "icu4c-data": {
+      "version": "0.67.2",
+      "resolved": "https://registry.npmjs.org/icu4c-data/-/icu4c-data-0.67.2.tgz",
+      "integrity": "sha512-OIRiop+k1IVf4TBLEOj910duoO9NKwtJLwp++qWT6KT5gRziHNt+5gwhcGuTqRy++RTK2gLoAIbk8KYCNxW++g==",
+      "dev": true
+    },
```

Comment:
You will want to want to revert these changes to package-lock and only include the changes to the redux version

Reply:
@fmunshi reverted the changed to `package-lock`